tests/testthat/test_get_axes.R

```diff
@@ -0,0 +1,14 @@
+context("get_axes")
+
+test_that("get x and y axes", {
+  p <- ggplot(iris, aes(Sepal.Length, Sepal.Width)) + geom_point()
+
+  expect_s3_class(get_x_axis(p), "gTree")
+  expect_s3_class(get_y_axis(p), "gTree")
+  expect_s3_class(get_x_axis(p, side = "t"), "zeroGrob")
+  expect_s3_class(get_y_axis(p, side = "r"), "zeroGrob")
+  expect_s3_class(get_y_axis(p + scale_y_continuous(position = "right"), side = "r"), "gTree")
```

Comment:
This example highlights the issue I mentioned above. You write `position = "right"` but `side = "r"`. It would be better to be consistent.

Reply:
Yes, I agree, so I'm switching the function to handle it internally (using just the first letter is fine with match.arg(). But it occurred to me, should the argument be called `position` instead of `side` to make that consistent, as well?

Reply:
I have no strong opinion on `position` vs `side`. As you prefer.

Reply:
I opted for `position` because I like the consistency.